### ISSUE TITLE
Disable warning when using with importmap-rails

### ISSUE THREAD

**Issue by phanhaiquang** [stimulusreflex/stimulus_reflex#563]
# Feature Request

Change checker from 'package.json' to 'config/importmap-rails' or using a custom config to define the file name of the file to check.

## Is your feature request related to a problem?

When using importmap-rails, config/importmap.rb is like below
pin 'stimulus_reflex', to: 'https://ga.jspm.io/npm:stimulus_reflex@3.4.1/javascript/stimulus_reflex.js'

Because importmap is used, we don't include this inside package.json. Due to this, it always show error

```
WARNING:
Can't locate the stimulus_reflex NPM package.
Either add it to your package.json as a dependency or use "yarn link stimulus_reflex" if you are doing development.
WARNING:
The Stimulus Reflex javascript package version () does not match the Rubygem version (3.4.1).
To update the Stimulus Reflex npm package:
    yarn upgrade stimulus_reflex@3.4.1
```

## Describe the solution you'd like

Maybe, we allow user to customize the way that this library is included in project ('package.json', or 'importmap',...) so that the checker could check the version at correct file.

## P/S

Thanks a lot for your great library !

**Comment by leastbad** [comment]
Hi there!

Rails 7 is still in alpha, and even trying to `rails new` on the current main branch is impossible. StimulusReflex and CableReady currently support Webpacker, ideally 5.4.3.

When Rails 7 is ready to be released, everything will be super smooth on our end. There will be extensive documentation and the setup script will lead people through the process.

We recommend using Rails 6.1.4.1 until Rails 7 is out of alpha. Alpha code in general is simply not ready for anything except working to get it ready for beta.

**Comment by phanhaiquang** [comment]
@leastbad thanks a lot for your comment. You are right, I could bypass this by adding corresponding package to package.json to temporary disable the warning.

**Comment by leastbad** [comment]
Have no doubt: we just want you to succeed. The many new JS asset strategies have been a big stress.

If you have any more questions: https://discord.gg/stimulus-reflex

**Comment by loqimean** [comment]
Hi there, it's still actual), and rails 7 aren't alpha and used in many projects! So, what about disabling this message if stimulus_reflex are present in an importmap file

**Comment by marcoroth** [comment]
Hey @loqimean, this has been resolved in the latest 3.5.0 pre-releases, so that shouldn't be an issue anymore. 3.5 supports all the major JavaScript bundling solutions Rails 7 supports.
